src/Diagnostics.ModelsAndUtils/Models/Storage/DiagConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Diagnostics.ModelsAndUtils.Models.Storage
+{
+    public class DiagConfiguration: TableEntity
+    {
+        public DiagConfiguration()
+        {
+
+        }
```

Comment:
Do you need this empty constructor?

Reply:
Yes, this is needed as `NullableStorageService` depends on it